rr_msgs/package.xml

```diff
@@ -3,6 +3,14 @@
 <package format="3">
   <name>rr_msgs</name>
   <version>0.0.0</version>
+<<<<<<< HEAD
+  <description>TODO: Package description</description>
```

Comment:
Think this is also in a few other places, but it looks like there are some places where GitHub is saying that both versions of merge conflicts are going to be added so make sure that the proper version Is being merged into main

Reply:
Oh oops some merge conflicts that I missed